### DISCUSSION TITLE
What is the size of the image

### DISCUSSION THREAD

**Discussion by nidiculageorge** [dotnet/dotnet-docker#5081] [Q&A]
Can we know the size of the image

**Comment by ghost** [comment]
I couldn't figure out the best area label to add to this issue. If you have write-permissions please help me learn by adding exactly one area label.

**Comment by lbussell** [comment]
Hi @nidiculageorge, in general, you can check the on-disk size of an image by running `docker images <your image name and tag>`. For example:

```
PS> docker images mcr.microsoft.com/dotnet/runtime:8.0
REPOSITORY                         TAG       IMAGE ID       CREATED       SIZE
mcr.microsoft.com/dotnet/runtime   8.0       141be24484c4   2 weeks ago   194MB
```

For a more general overview of deployment methods and resulting image sizes to help you choose which image to use, please check out our [Sample Image Size Report](https://github.com/dotnet/dotnet-docker/blob/main/documentation/sample-image-size-report.md).

**Comment by nidiculageorge** [comment]
Hi @lbussell 

I am new to the docker stuff and working on security side . We are working on finalizing the base image for our application, currently our application is running in 6 and getting migrated to  dotnet 8

The current base image is running using debian and the base image is below.

mcr.microsoft.com/dotnet/runtime:6.0

Queries:

If we are finalizing a Base Image which supports 8.0 for example as below

mcr.microsoft.com/dotnet/sdk:8.0 

1. How often does a security fix is released for these images
2. How can we mention a new base image for the above sdk  to a new base image tag
